classes/class-send-products.php

```diff
@@ -105,6 +104,71 @@ public static function get_product_type_price($product)
             );
         }
 
+
+        /**
+         * Checks if stock management is enabled and if so, returns quantity and status
+         *
+         * @param  mixed $product Product to check   
+         * @return array ['stock_quantity' => $stock_quantity,'stock_status' => $stock_status] Array with quantity and status. if stock management is disabled, false will be returned,
+         */
+        public static function get_product_stock_data($product)
+        {
+            if ($product->get_manage_stock()) {
+                return array(
+                    'stock_quantity' => $product->get_stock_quantity(),
```

Comment:
@SublimeProphets 

Does this handle all type of products, like variable products and simple products?

Reply:
yes, it works for both variable and simple products. it does always retrieve the data from the main tab, the data for variable products is not used.